src/Components/FashCard/FlashCardQuestions/CodeBlockStyle/CodeBlockStyle.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import { Question } from "../../../../Apis/types";
+import CodeEditor from "@uiw/react-textarea-code-editor";
+
+interface Props {
+    cardQuestion: Question;
+}
+
+export const CodeBlockStyle = ({ cardQuestion }: Props) => {
+    const [code, setCode] = useState<string>(
+        `const array = [-3, -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13];
+
+        function isPrime(num) {
+          for (let i = 2; num > i; i++) {
+            if (num % i === 0) {
+              return false;
+            }
+          }
+          return num > 1;
+        }
+
+        console.log(array.filter(isPrime)); // [2, 3, 5, 7, 11, 13]`
+    );
```

Comment:
this is just simple hard codded, this will be the format that is sent the component though from the question.